kernel/src/main/scala/org/apache/toree/kernel/api/Kernel.scala

```diff
@@ -396,7 +372,29 @@ class Kernel (
     interpreterManager.interpreters.get(name)
   }
 
-  override def sparkSession: SparkSession = SparkSession.builder.getOrCreate
+  private lazy val defaultSparkConf: SparkConf = createSparkConf(
+    new SparkConf().setAppName(SparkKernelInfo.banner))
```

Comment:
Btw, I think your build issue can be resolved via `git clean -xdf`.  This came up in the chat earlier: https://gitter.im/apache/toree?at=58c6cd0509e7ba8510bd50b0  I believe that's a one time thing following Jakob's sbt changes.